Guitars_2019-08-06.sql

```diff
@@ -0,0 +1,202 @@
+	(12,'Guild'),
+	(13,'Yamaha'),
+	(14,'Washburn'),
+	(15,'Ovation'),
+	(16,'Guild'),
+	(17,'Crafter'),
+	(18,'Vintage');
+
+/*!40000 ALTER TABLE `brands` ENABLE KEYS */;
+UNLOCK TABLES;
+
+
+# Dump of table countries
+# ------------------------------------------------------------
+
+DROP TABLE IF EXISTS `countries`;
+
+CREATE TABLE `countries` (
+  `id` int(11) unsigned NOT NULL AUTO_INCREMENT,
+  `country` varchar(255) NOT NULL DEFAULT '',
+  PRIMARY KEY (`id`)
+) ENGINE=InnoDB DEFAULT CHARSET=latin1;
+
+LOCK TABLES `countries` WRITE;
+/*!40000 ALTER TABLE `countries` DISABLE KEYS */;
+
+INSERT INTO `countries` (`id`, `country`)
+VALUES
+	(1,'Germany'),
+	(2,'USA'),
+	(3,'Canada'),
+	(4,'Japan'),
+	(5,'France'),
+	(6,'China'),
+	(7,'UK'),
+	(8,'India');
+
+/*!40000 ALTER TABLE `countries` ENABLE KEYS */;
+UNLOCK TABLES;
+
+
+# Dump of table guitars
+# ------------------------------------------------------------
+
+DROP TABLE IF EXISTS `guitars`;
+
+CREATE TABLE `guitars` (
+  `id` int(11) unsigned NOT NULL AUTO_INCREMENT,
+  `brandID` int(11) unsigned NOT NULL,
+  `model` varchar(255) NOT NULL DEFAULT '',
+  `year` year(4) DEFAULT NULL,
+  `typeID` int(11) unsigned NOT NULL,
+  `countryID` int(11) unsigned NOT NULL,
+  `LH or RH` enum('LH','RH') NOT NULL,
+  `value` int(11) DEFAULT NULL,
+  `serialCode` varchar(255) DEFAULT NULL,
+  `dateAcquired` date NOT NULL,
+  PRIMARY KEY (`id`),
+  KEY `brand` (`brandID`),
+  KEY `countries` (`countryID`),
+  KEY `type` (`typeID`),
+  CONSTRAINT `brand` FOREIGN KEY (`brandID`) REFERENCES `brands` (`id`),
+  CONSTRAINT `countries` FOREIGN KEY (`countryID`) REFERENCES `countries` (`id`),
+  CONSTRAINT `type` FOREIGN KEY (`typeID`) REFERENCES `types` (`id`)
+) ENGINE=InnoDB DEFAULT CHARSET=latin1;
+
+LOCK TABLES `guitars` WRITE;
+/*!40000 ALTER TABLE `guitars` DISABLE KEYS */;
+
+INSERT INTO `guitars` (`id`, `brandID`, `model`, `year`, `typeID`, `countryID`, `LH or RH`, `value`, `serialCode`, `dateAcquired`)
+VALUES
+	(1,3,'Les Paul Classic, Gold Top','2019',1,2,'RH',1649,NULL,'2019-08-05'),
+	(2,1,'Stratocaster','1983',1,4,'RH',340,'E432-Q34ER3','2017-02-14'),
+	(3,4,'Dot 335, Cherry Red','2004',1,1,'RH',349,'ED04-23486','2005-04-16'),
+	(4,8,'GSR205, Walnut Flat','2016',3,5,'RH',269,'105279','2016-05-26'),
+	(5,4,'Casino, Sunburst','1964',1,2,'RH',4000,'1345543-EC64','1978-03-19'),
+	(6,18,'ElectroAcoustic, Natural Satin','2017',3,7,'RH',399,'17EAB-VNS491','2018-05-14'),
+	(7,7,'4003LE, Pillar Box Red','1996',3,1,'RH',2739,'139408','2001-10-18'),
+	(8,5,'G5420T Electromatic, Aspen Green','1958',1,3,'RH',879,'GEAG-23512','1987-11-01');
+
+/*!40000 ALTER TABLE `guitars` ENABLE KEYS */;
+UNLOCK TABLES;
+
+
+# Dump of table images
+# ------------------------------------------------------------
+
+DROP TABLE IF EXISTS `images`;
+
+CREATE TABLE `images` (
+  `id` int(11) unsigned NOT NULL AUTO_INCREMENT,
+  `guitarID` int(11) unsigned DEFAULT NULL,
+  `fileLocation` varchar(255) NOT NULL DEFAULT '',
```

Comment:
Should maybe allow nulls for when you're adding new entries later on and might have to add image seperatly?

Reply:
or could default to a placeholder image?